docs/tooling/flow-cli/flix.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Execute Flow Interaction Templates (FLIX) with the Flow CLI
+sidebar_title: Execute FLIX
+description: How to execute Flow Interaction Templates (FLIX) on Flow from the command line
+sidebar_position: 12
+---
+
+The Flow CLI provides a command to execute Flow Interaction Templates (FLIX).
+
+```shell
+flow flix <query> [<argument> <argument>...] [flags]
+```
+
+Queries can be a FLIX id, name, or path to a local FLIX file.
+
+## Example Usage
+
+```shell
+# Execute a FLIX script by id on Testnet
```

Comment:
very NIT: but I feel the example with name should go first as it feels more natural to understand the command. Also the ID should be in correct format hex encoded ID